projects/aws/eks-charts/aws-for-fluent-bit/Makefile

```
@@ -0,0 +1,42 @@
+AWS_FOR_FLUENT_BIT_CHART_REPO_URL?=https://github.com/aws/eks-charts/
+AWS_FOR_FLUENT_BIT_CHART_REPO?=eks-charts
+AWS_FOR_FLUENT_BIT_CHART_GIT_TAG?=v0.0.41
+
+ifeq ("$(AWS_FOR_FLUENT_BIT_CHART_REPO_URL)","")
+        $(error No repository URL was provided.)
+endif
+
+ifeq ("$(AWS_FOR_FLUENT_BIT_CHART_REPO)","")
+	$(error No repository name was provided.)
+endif
+
+ifeq ("$(AWS_FOR_FLUENT_BIT_CHART_GIT_TAG)","")
+	$(error No gittag was provided.)
+endif
+
+CHART_SCRIPTS_DIR=$(shell git rev-parse --show-toplevel)/helm-charts/scripts
+MAKE_ROOT=$(shell cd "$(dirname "$(BASH_SOURCE[0])")" && pwd -P)
+CLONE_ROOT=$(MAKE_ROOT)/$(AWS_FOR_FLUENT_BIT_CHART_REPO)
+
+.PHONY: clone
+clone: clean
+	git clone  $(AWS_FOR_FLUENT_BIT_CHART_REPO_URL) $(AWS_FOR_FLUENT_BIT_CHART_REPO)
```

nit: 1 extra space after `clone`
```suggestion
	git clone $(AWS_FOR_FLUENT_BIT_CHART_REPO_URL) $(AWS_FOR_FLUENT_BIT_CHART_REPO)
```